lib/tasks/one_off.rake

```diff
@@ -223,8 +223,8 @@ namespace :pvb do
     queue = Queue.new
     task = ->(client, visit) { check_slot_availability(client, visit) }
 
-    visits = Visit.select('visits.*', 'estates.nomis_id nomis_id').
```

Comment:
I believe the `AS` is optional, or it was until recently. I suspect a change of behaviour in ActiveRecord